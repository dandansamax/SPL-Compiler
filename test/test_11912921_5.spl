char test_11912921_5(){
    int c[5];
    if(c[1]>=2){
        return c[0];
        
    }
    else{
        return c[1]; 

    }
}