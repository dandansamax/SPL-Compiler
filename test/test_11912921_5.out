Error type 4 at Line 5: a function is redefined: redef
